modules/nf-core/parabricks/applybqsr/tests/main.nf.test

```diff
@@ -0,0 +1,101 @@
+nextflow_process {
+
+    name "Test Process PARABRICKS_APPLYBQSR"
+    script "../main.nf"
+    process "PARABRICKS_APPLYBQSR"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "parabricks"
+    tag "parabricks/applybqsr"
+
+    test("parabricks_applybqsr") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test'],
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
+                    [], // index not needed unless using intervals
+                    file(params.test_data['sarscov2']['illumina']['test_baserecalibrator_table'], checkIfExists: true),
```

Comment:
we need to update the way these paths are represented
-> look at other modules but it is something along the line of
`file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/vcf/test.vcf.gz', checkIfExists: true)`
You can check https://github.com/nf-core/modules/blob/master/tests/config/test_data.config to find the correct paths.

Reply:
Hope it's okay now